packages/account/__mocks__/translation.mock.js

```diff
@@ -0,0 +1,17 @@
+const Localize = ({ i18n_default_text, values }) => {
```

Comment:
[Question] - is this only for a testing purpose?

Reply:
Yes. This way we are mocking the TranslationProvider globally so the hook useTranslation can be configured as required within test suits